harvard-lts/CURIOSity
# Push to Harvard 11 22 2022

**Push to Harvard 11 22 2022**
* * *

**JIRA Ticket**: 
* https://github.com/harvard-lts/CURIOSity/issues/229
* https://github.com/harvard-lts/CURIOSity/issues/254
* https://github.com/harvard-lts/CURIOSity/issues/217
* https://github.com/harvard-lts/CURIOSity/issues/196
* https://github.com/harvard-lts/CURIOSity/issues/77

# What does this Pull Request do?
Incorporates everything in the tickets above, as well as a fix to the ENV template to support the new docker-compose and adding in scientist GitHub actions that we won't be using but are in the code (shouldn't fire in our GA).

# How should this be tested?

To test everything but Github Actions, pull this branch and update your env file using the env-example. Rebuild the container and look and see if you see all the changes in the UI tickets above.

GA will need to be reviewed in the GA console.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
